### PR TITLE
Header

### PR DESCRIPTION
**Pull request by karinaramos0401**

### CODE REVIEW

File: css/_reset.css [hunk 1]
```diff
@@ -49,4 +50,5 @@ table {
 
 * {
     box-sizing: border-box;
+    
   }
```
Comment (juunegreiros) [inline css/_reset.css:54]: Can you add a new line here?

File: css/index.css [hunk 2]
```diff
@@ -1,3 +1,4 @@
 @import "_reset.css";
 @import "_variables.css";
-
+@import "_header.css";
+@import "_structure.css";
```
Comment (juunegreiros) [inline css/index.css:4]: Can you add a newline here too?

File: index.html [hunk 3]
```diff
@@ -7,10 +7,21 @@
...
+            <h1 class="title">K A R I N A  R A M O S </h1>
+        </div>
+        <div class="aside">hahaha</div>
+    </div>
+
+
+   
+    <footer class="footer">###</footer>
 </body>
 </html>
```
Comment (juunegreiros) [inline index.html:27]: And here

File: index.html [hunk 4]
```diff
@@ -7,10 +7,21 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@300;400&display=swap" rel="stylesheet">
 </head>
 <body>
-    <h1>Karina Ramos</h1>
+
+    <div class="container"> 
+        <div class="about-me">
+            <h1 class="title">K A R I N A  R A M O S </h1>
```
Comment (juunegreiros) [inline index.html:18]: Can you write this as "Karina Ramos"? You can fix this with css adding 

```css
text-transform: uppercase;
word-spacing: *your value*;
```

File: css/_header.css [hunk 5]
```diff
@@ -0,0 +1,23 @@
+
+.title{ 
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-family: var(--primary-font);
+    font-size: 45px;
+    position: relative;
+}
+
```
Comment (lahgomes) [inline css/_header.css:10]: Can you add more margin top in your title?

**Pull request merged**